#! /bin/sh /usr/share/dpatch/dpatch-run
## 00_01_upstream_0.9.11-1.dpatch by HPLIP Upstream
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: This is a pre-patch (not the official 0.9.11-1 patch)
## DP: - fix regression on the PSC 950xi

@DPATCH@
diff -urNad hplip~/data/xml/models.xml hplip/data/xml/models.xml
--- hplip~/data/xml/models.xml	2006-05-10 00:37:19.000000000 -0300
+++ hplip/data/xml/models.xml	2006-05-10 21:32:37.912772306 -0300
@@ -7797,6 +7797,32 @@
     </r0>
 </model>
 
+<model name="PSC_950xi">
+    <id>MFG:Hewlett-Packard;MDL:PSC 900 Series;SKU:PSC 950;CMD:2,PCL,PML;CLASS:PRINTER;DESCRIPTION:Hewlett-Packard PSC 900 Series;1284.3M:f7f,f7f;1284.4DL:4d,4e,1;SERN:MY15U31143WP;VSTATUS:$AU0$AU0,ff,DN,PENF,CUT,K0,C0,SM,NR,KP066,CP095;AiO:00;DW-PCL;</id>
+    <icon>PSC_900_Series.png</icon>
+    <support type="2"/>
+    <case-model>
+    </case-model>
+    <io support="2" mode="1" control="0" mfp-mode="2" scan-port="0"/>
+    <align type="9"/>
+    <clean type="3"/>
+    <color-cal type="0"/>
+    <status type="1"/>
+    <scan type="1"/>
+    <fax type="0"/>
+    <pcard type="1"/>
+    <copy type="0"/>
+    <tech class="DJ9xx" type="2"/>
+    <embedded-server type="0"/>
+    <panel-check type="1"/>
+    <linefeed-cal type="0"/>
+    <pq-diag type="0"/>
+    <r0>
+        <agent1 kind="3" type="1" sku="15 (C6615DN)"/>
+        <agent2 kind="3" type="2" sku="78 (C6578AN / C6578DN)"/>
+    </r0>
+</model>
+
 <model name="Photosmart_2570_series">
     <id></id>
     <icon>psc_2300_series.png</icon>
